app/forms/auth.py

```diff
@@ -41,3 +41,11 @@ def validate_username(form, field):
     def validate_email(form, field):
         if User.query.filter_by(email=field.data).first() is not None:
             raise ValidationError("This email is already registered.")
+
+
+class EditUserForm(RegistrationForm):
+    def validate_username(form, field):
+        pass
+
+    def validate_email(form, field):
+        pass
```

Comment:
```
class EditUserForm(RegistrationForm):
    def validate_username(form, field):
        pass

    def validate_email(form, field):
        pass
```
Is this code should be empty?

Reply:
I wanted to said "Yes", but found the bug in this logic... Thank you!